### PR TITLE
new icon: dropwizard (original, plain)

### PR DESCRIPTION
**Pull request by Mazzya**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date
* [Official website](https://www.dropwizard.io/en/latest/)
* [Official github repository](https://github.com/dropwizard/dropwizard)

## This PR adds/fixes
* Closes [Icon request: dropwizard #891 ](https://github.com/devicons/devicon/issues/891)

### CODE REVIEW

File: icons/dropwizard/dropwizard-original.svg [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+</g>
+</g>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(97.647059%,65.490196%,16.862745%);fill-opacity:1;" d="M 37.042969 97.6875 L 42.605469 91.824219 L 47.800781 98 L 41.855469 105.964844 L 37.042969 97.6875 M 82.460938 75.496094 L 88.019531 69.632812 L 93.21875 75.808594 L 87.269531 83.777344 L 82.460938 75.496094 M 51.367188 52.082031 L 56.925781 46.21875 L 62.121094 52.398438 L 56.175781 60.359375 L 51.367188 52.082031 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,94.901961%,0%);fill-opacity:1;" d="M 37.042969 97.6875 L 42.613281 90.820312 L 47.800781 98 L 41.863281 103.871094 L 37.042969 97.6875 M 82.460938 75.496094 L 88.03125 68.632812 L 93.21875 75.808594 L 87.28125 81.683594 L 82.460938 75.496094 M 51.367188 52.082031 L 56.9375 45.21875 L 62.121094 52.398438 L 56.183594 58.269531 L 51.367188 52.082031 "/>
+<path style=" stroke:none;fill-rule:nonzero;fill:rgb(14.117647%,14.901961%,36.470588%);fill-opacity:1;" d="M 42.585938 113.742188 C 42.585938 113.742188 82.425781 117.804688 108.648438 107.546875 C 105.359375 113.742188 75.957031 120.804688 42.585938 113.742188 "/>
+</g>
+</svg>
```
Comment (Panquesito7) [inline icons/dropwizard/dropwizard-original.svg:44]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><defs><clipPath id="a"><path d="M9 0h112v126H9Zm0 0"/></clipPath><clipPath id="b"><path d="M9.926 118.5s62.547 18.965 110.402-3.574C107.848 94.46 85.938 35.418 72.996 9.344 70.238 3.785 69.79 3.23 68.66 1.707c-6.316-8.543-12.96 16.57-23.765 41.852C41.023 52.62 9.925 118.5 9.925 118.5Zm0 0"/></clipPath><clipPath id="d"><path d="M22 2h99v122H22Zm0 0"/></clipPath><clipPath id="e"><path d="M22.543 114.309s53.508 19.183 97.785.617c-1.183-3.235-4.207-8.422-6.258-13.254C103.172 76 83.441 32.652 73.828 11.129c-10.418-23.3-13.789 3.945-25.941 32.937-3.696 8.817-25.344 70.243-25.344 70.243Zm0 0"/></clipPath><linearGradient id="c" gradientUnits="userSpaceOnUse" x1="0" y1="0" x2="1" y2="0" gradientTransform="matrix(-168.46739 54.6522 -89.44685 -102.9339 176.764 30.713)"><stop offset="0" style="stop-color:#3871c1;stop-opacity:1"/><stop offset=".515" style="stop-color:#2c3792;stop-opacity:1"/><stop offset=".865" style="stop-color:#24265d;stop-opacity:1"/><stop offset="1" style="stop-color:#252761;stop-opacity:1"/></linearGradient><linearGradient id="f" gradientUnits="userSpaceOnUse" x1="0" y1="0" x2="1" y2="0" gradientTransform="matrix(-98.31357 15.67052 -25.64725 -60.06978 95.252 59.744)"><stop offset="0" style="stop-color:#3871c1;stop-opacity:1"/><stop offset=".515" style="stop-color:#2c3792;stop-opacity:1"/><stop offset=".865" style="stop-color:#24265d;stop-opacity:1"/><stop offset="1" style="stop-color:#252761;stop-opacity:1"/></linearGradient></defs><g clip-path="url(#a)"><g clip-path="url(#b)"><path style="stroke:none;fill-rule:nonzero;fill:url(#c)" d="M9.926 118.5s62.547 18.965 110.402-3.574C107.848 94.46 85.938 35.418 72.996 9.344 70.238 3.785 69.79 3.23 68.66 1.707c-6.316-8.543-12.96 16.57-23.765 41.852C41.023 52.62 9.925 118.5 9.925 118.5"/></g></g><g clip-path="url(#d)"><g clip-path="url(#e)"><path style="stroke:none;fill-rule:nonzero;fill:url(#f)" d="M22.543 114.309s53.508 19.183 97.785.617c-1.183-3.235-4.207-8.422-6.258-13.254C103.172 76 83.441 32.652 73.828 11.129c-10.418-23.3-13.789 3.945-25.941 32.937-3.696 8.817-25.344 70.243-25.344 70.243"/></g></g><path style="stroke:none;fill-rule:nonzero;fill:#f9a72b;fill-opacity:1" d="m37.043 97.688 5.562-5.864L47.801 98l-5.946 7.965-4.812-8.278m45.418-22.19 5.559-5.864 5.199 6.176-5.95 7.968-4.808-8.28M51.367 52.081l5.559-5.863 5.195 6.18-5.945 7.96-4.809-8.277"/><path style="stroke:none;fill-rule:nonzero;fill:#fff200;fill-opacity:1" d="m37.043 97.688 5.57-6.868L47.801 98l-5.938 5.871-4.82-6.183m45.418-22.192 5.57-6.863 5.188 7.176-5.938 5.875-4.82-6.188M51.367 52.082l5.57-6.863 5.184 7.18-5.937 5.87-4.817-6.187"/><path style="stroke:none;fill-rule:nonzero;fill:#24265d;fill-opacity:1" d="M42.586 113.742s39.84 4.063 66.062-6.195c-3.289 6.195-32.691 13.258-66.062 6.195"/></svg>

```

**Comment by Panquesito7** [comment]
The icon is great, good work! But I think the original version won't fit the Icomoon version.
Please create a `plain` version that has only *one* color and the sparkles transparent.

Let us know if you need any help. Thanks. 😄

**Comment by Mazzya** [comment]
Done 🚀

File: icons/dropwizard/dropwizard-original.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (Panquesito7) [inline icons/dropwizard/dropwizard-original.svg:2]:
```suggestion
```

File: icons/dropwizard/dropwizard-plain.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+</g>
+</svg>
```
Comment (Panquesito7) [inline icons/dropwizard/dropwizard-plain.svg:6]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128"><path style="stroke:none;fill-rule:nonzero;fill:#000;fill-opacity:1" d="M64 .77c-2.785 1.8-5.46 7.312-12.941 26.52-6.282 16-5.133 13.429-23.317 52.51l-14.91 31.833-2.84.77c-20.586 5.577-5.844 12.425 32.055 14.929 8.683.566 35.824.566 43.96 0 20.423-1.437 33.692-3.809 39.153-7.016 1.094-.632 2.184-1.601 2.348-2.136.492-1.371-1.418-3.04-5.133-4.442l-3.113-1.172-2.293-4.773c-1.254-2.606-5.461-12.027-9.336-20.945-3.88-8.918-10.649-24.184-15.02-33.903-4.367-9.754-9.773-21.914-12.011-27.058-5.133-11.758-9.668-21.012-11.524-23.45C67.113-.233 66.02-.566 64 .77Zm-4.531 48.6c2.676 3.34 2.73 2.938-.875 7.68-1.692 2.304-2.567 3.074-2.895 2.742-.219-.27-1.308-2.105-2.457-4.078l-2.02-3.54 2.622-2.874c1.472-1.567 2.785-2.871 2.949-2.871.164 0 1.363 1.336 2.676 2.941Zm31.18 23.649 2.402 2.738-2.95 3.875c-1.636 2.137-3.113 3.773-3.277 3.64-.543-.402-4.367-7.082-4.367-7.648 0-.77 5.023-5.879 5.461-5.578.219.133 1.473 1.504 2.73 2.973ZM45.323 94.898c1.203 1.438 2.184 2.774 2.184 2.973 0 .168-1.254 2.07-2.84 4.176-3.223 4.441-2.621 4.543-5.898-1.238l-1.801-3.207 2.62-2.872c1.477-1.57 2.84-2.773 3.114-2.671.274.101 1.418 1.37 2.621 2.84Zm0 0"/></svg>

```

File: icons/dropwizard/dropwizard-plain.svg [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+
```
Comment (Panquesito7) [inline icons/dropwizard/dropwizard-plain.svg:2]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path style="stroke:none;fill-rule:nonzero;fill:#000;fill-opacity:1" d="M64 .77c-2.785 1.8-5.46 7.312-12.941 26.52-6.282 16-5.133 13.429-23.317 52.51l-14.91 31.833-2.84.77c-20.586 5.577-5.844 12.425 32.055 14.929 8.683.566 35.824.566 43.96 0 20.423-1.437 33.692-3.809 39.153-7.016 1.094-.632 2.184-1.601 2.348-2.136.492-1.371-1.418-3.04-5.133-4.442l-3.113-1.172-2.293-4.773c-1.254-2.606-5.461-12.027-9.336-20.945-3.88-8.918-10.649-24.184-15.02-33.903-4.367-9.754-9.773-21.914-12.011-27.058-5.133-11.758-9.668-21.012-11.524-23.45C67.113-.233 66.02-.566 64 .77Zm-4.531 48.6c2.676 3.34 2.73 2.938-.875 7.68-1.692 2.304-2.567 3.074-2.895 2.742-.219-.27-1.308-2.105-2.457-4.078l-2.02-3.54 2.622-2.874c1.472-1.567 2.785-2.871 2.949-2.871.164 0 1.363 1.336 2.676 2.941Zm31.18 23.649 2.402 2.738-2.95 3.875c-1.636 2.137-3.113 3.773-3.277 3.64-.543-.402-4.367-7.082-4.367-7.648 0-.77 5.023-5.879 5.461-5.578.219.133 1.473 1.504 2.73 2.973ZM45.323 94.898c1.203 1.438 2.184 2.774 2.184 2.973 0 .168-1.254 2.07-2.84 4.176-3.223 4.441-2.621 4.543-5.898-1.238l-1.801-3.207 2.62-2.872c1.477-1.57 2.84-2.773 3.114-2.671.274.101 1.418 1.37 2.621 2.84Zm0 0"/></svg>
```

**Review by Panquesito7 — DISMISSED** [review]
Awesome! 🚀 Thank you for your contribution! 😄👍

**Comment by Thomas-Boi** [comment]
Hi @Mazzya.

Thank you for the icon but it looks different compared to the company's official logo.

Here is the company's version:
image

Here's your version:
image

The company's version is skinnier than yours. I think you accidentally change the icon's width when you change its `width` and `height`.  Can you edit the image so it looks more similar to the company's version please?

**Review by Thomas-Boi — CHANGES_REQUESTED** [review]
Please change both the `plain` and the `original` version. Thank you

**Comment by Thomas-Boi** [comment]
I'll close this PR since seems like there's no activity for now.

**Comment by Panquesito7** [comment]
What if we update this ourselves? @Thomas-Boi.

**Comment by Thomas-Boi** [comment]
@Panquesito7 That's fine too as long as someone volunteers. If you are interested, feel free to reopen and assign yourself to the PR 😃

**Comment by Panquesito7** [comment]
I'll try to fix it on these days. Thanks.

**Comment by Thomas-Boi** [comment]
Hey @Mazzya @Panquesito7, anyone still wanting to work on this PR? If not, we can start making preparations for the next release.

**Comment by Thomas-Boi** [comment]
@Panquesito7 good news, the Dropwizard logo is actually an SVG already: 
image

If you are still interested, you can just download it and resize the canvas/document (not the logo itself) and simplify it for the `plain` version. Else, I can do it too but probably nearer to the release date

**Review by Thomas-Boi — APPROVED** [review]

**Review by Thomas-Boi — APPROVED** [review]
Updated the svg files. Will delete eps file in a separate PR

**Pull request merged**